intern/frontend/src/index.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import { Cart } from './components/Cart/Cart'
+import { Shop } from './components/Shop/Shop'
+
+const App = () => {
+    const [products, setProducts] = React.useState([])
+    const [availableProducts, setAvailableProducts] = React.useState([])
+    const [inCartProducts, setInCartProducts] = React.useState([])
+
+    React.useEffect(() => {
```

Comment:
maybe add a comment to explain that we are fetching products from the backend and storing them in the products state ?